client/package.json

```diff
@@ -30,5 +30,11 @@
             "last 1 firefox version",
             "last 1 safari version"
         ]
+    },
+    "devDependencies": {
```

Comment:
These four should be standard dependencies because they'll be needed for deployment